test/references/retry_tests_report_1worker.xml

```diff
@@ -0,0 +1,328 @@
+  Expression: false
+		</error>
+	</testcase>
+	<testcase name="Timeout always" timestamp="2023-06-21T13:40:14.684" time="0.3" tests="1" skipped="0" failures="0" errors="1">
+		<properties>
+		<property name="dd_tags[perf.bytes]" value="0"></property>
+		<property name="dd_tags[perf.allocs]" value="0"></property>
+		<property name="dd_tags[perf.gctime]" value="0.0"></property>
+		<property name="dd_tags[perf.compile_time]" value="0.0"></property>
+		<property name="dd_tags[perf.recompile_time]" value="0.0"></property>
+		<property name="dd_tags[perf.eval_time]" value="0.0"></property>
+		</properties>
+		<error message="Error during test at test/testfiles/_retry_tests.jl:52">Captured Logs for test item "Timeout always" at test/testfiles/_retry_tests.jl:52 on worker 51271
+
+signal (15): Terminated: 15
+in expression starting at /Users/nickr/repos/ReTestItems.jl/test/testfiles/_retry_tests.jl:58
+__psynch_cvwait at /usr/lib/system/libsystem_kernel.dylib (unknown line)
+unknown function (ip: 0x0)
+kevent at /usr/lib/system/libsystem_kernel.dylib (unknown line)
+unknown function (ip: 0x0)
+Allocations: 5228932 (Pool: 5226603; Big: 2329); GC: 30
+Error in testset "Timeout always" on worker 51271:
+Error During Test at /Users/nickr/repos/ReTestItems.jl/test/testfiles/_retry_tests.jl:52
+  Got exception outside of a @test
+  Timed out after 5s evaluating test item "Timeout always" (run=1)
+		</error>
+	</testcase>
+	<testcase name="Timeout always" timestamp="2023-06-21T13:40:20.82" time="0.0" tests="1" skipped="0" failures="0" errors="1">
+		<properties>
+		<property name="dd_tags[perf.bytes]" value="0"></property>
+		<property name="dd_tags[perf.allocs]" value="0"></property>
+		<property name="dd_tags[perf.gctime]" value="0.0"></property>
+		<property name="dd_tags[perf.compile_time]" value="0.0"></property>
+		<property name="dd_tags[perf.recompile_time]" value="0.0"></property>
+		<property name="dd_tags[perf.eval_time]" value="0.0"></property>
+		</properties>
+		<error message="Error during test at test/testfiles/_retry_tests.jl:52">Captured Logs for test item "Timeout always" at test/testfiles/_retry_tests.jl:52 on worker 51271
+
+signal (15): Terminated: 15
+in expression starting at /Users/nickr/repos/ReTestItems.jl/test/testfiles/_retry_tests.jl:58
+__psynch_cvwait at /usr/lib/system/libsystem_kernel.dylib (unknown line)
+unknown function (ip: 0x0)
+kevent at /usr/lib/system/libsystem_kernel.dylib (unknown line)
+unknown function (ip: 0x0)
+Allocations: 3068788 (Pool: 3067677; Big: 1111); GC: 3
+Error in testset "Timeout always" on worker 51271:
+Error During Test at /Users/nickr/repos/ReTestItems.jl/test/testfiles/_retry_tests.jl:52
+  Got exception outside of a @test
+  Timed out after 5s evaluating test item "Timeout always" (run=2)
+		</error>
+	</testcase>
+	<testcase name="Timeout always" timestamp="2023-06-21T13:40:26.634" time="0.0" tests="1" skipped="0" failures="0" errors="1">
```

Comment:
All 3 runs of `Timeout always` should be here, i.e. there should be 3 `<testcase name="Timeout always"` entries